mobile/lib/widgets/error_widgets.dart

```diff
@@ -559,8 +559,8 @@ class AppCrushWidget extends StatelessWidget {
             const Spacer(),
             InkWell(
               onTap: () async {
+                // TODO log to  a backend service
                 PackageInfo packageInfo = await PackageInfo.fromPlatform();
-                String email = "support@airqo.net";
```

Comment:
Yeah, thanks @NoahNsimbe for this removal.
I will set up a special notification tool for all of us, soon. 
Let us first get done with Access Control.